ansible_collections/arista/avd/roles/eos_designs/doc/fabric-topology.md

```diff
@@ -345,6 +345,48 @@ defaults <- node_group <- node_group.node <- node
         inter_domain: < true | false | Default -> True >
 ```
 
+### BGP & MPLS-VPN Gateway
+
+```yaml
+< node_type_key >:
+
+  defaults:
+    # Node is acting as MPLS-VPN Gateway for EVPN to MPLS-VPN Interworking | Optional.
+    # A new BGP peer-group is used for this neighborship -> "bgp_peer_groups.mpls_vpn_gateway_overlay_peers".
+    # L3 Reachability is required for this to work, the preferred method to establish underlay connectivity is to use core_interfaces.
```

Comment:
You could add something like 
"requires `evpn_role` _and_ `mpls_overlay_role` to be set"
to simplify a lot of the logic. Checks for these could be included in the fact function.